test/functional/rpc_deprecated.py

```diff
@@ -4,13 +4,13 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test deprecation of RPC calls."""
 from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import assert_raises_rpc_error
+# from test_framework.util import assert_raises_rpc_error
```

Comment:
Rather than change this file around, can you just revert it to how it was pre-commit c9f02955b2e9062808a9455c4ee7d52cf401eef5.

Reply:
Main reason for changing this file around is improving documentation; `generate` allows a much simpler example than `createmultisig`.